server/cmd/dcrdex/main.go

```diff
@@ -78,6 +79,19 @@ func mainCore(ctx context.Context) error {
 	log.Infof("Found %d assets, loaded %d markets, for network %s",
 		len(assets), len(markets), strings.ToUpper(cfg.Network.String()))
 
+	// Load, or create and save, the DEX signing key.
+	var privKey *secp256k1.PrivateKey
+	{
+		keyPW, err := admin.PasswordPrompt("Signing key password: ")
```

Comment:
I'm a little nervous about password security.  Does it sit in memory?  Can we clear a string in place?  When GC gets it (the scope I made here), does the OS clear it or otherwise make it inaccessible to other processes that might need the physical memory that was returned to the OS?

Reply:
Probably makes sense to change to `[]byte` wherever possible so we can manually zero. Looks like `PasswordPrompt` is converting to `string`, which leaves us at the mercy of the garbage collector. The **encrypt** functions and methods take a string, but that's an easy change.

Reply:
Right, I didn't want to open the can of worms with the encrypt API.  I suppose we can hack on encrypt in a followup.

Reply:
https://github.com/decred/dcrdex/pull/259/commits/eeab104ef2b16ea92b70d5f367452506fa5d802b changes password handling in admin and dcrdex to a `[]byte` instead, but converts to `string` on call to `encrypt.Deserialize` and `encrypt.NewCrypter`, where there are TODOs for modifying `encrypt`.